### ISSUE TITLE
Use Ahab libs from Nexus

### ISSUE THREAD

**Issue by hinchliff** [RENCI-NRIG/Mobius#1]
When I checkout the project, and try to build, I get the following maven error:

>[ERROR] Failed to execute goal on project RequestManager: Could not resolve dependencies for project org.renci:RequestManager:jar:1.0-SNAPSHOT: The following artifacts could not be resolved: orca.ahab:libtransport:jar:0.1-SNAPSHOT, orca.ahab:libndl:jar:0.1-SNAPSHOT: Could not find artifact orca.ahab:libtransport:jar:0.1-SNAPSHOT in geni-orca-snapshot (http://ci-dev.renci.org/nexus/content/repositories/geni-orca-snapshot)

Those old ahab artifacts are no longer available.

Our first step should be to get the project to build using the new [Ahab release](https://github.com/RENCI-NRIG/ahab) artifacts.

**Comment by hinchliff** [comment]
@anriban I created a new branch for this issue, [1_ahab_release](https://github.com/RENCI-NRIG/Mobius/tree/1_ahab_release), but it does not currently compile.

I'm not sure if the Ahab library changed?  But I get the error "no suitable method found"

>[ERROR] /home/hincha/git/Mobius/src/main/java/org/renci/requestmanager/ndl/AhabManager.java:[372,54] no suitable method found for create(org.renci.ahab.libndl.Slice,java.lang.String)
[ERROR] method org.renci.ahab.libndl.extras.PriorityNetwork.create(org.renci.ahab.libndl.Slice,java.lang.String,java.lang.String,long) is not applicable
[ERROR] (actual and formal argument lists differ in length)

Could you take a look at this?

Possibly related, is the `orca.ndllib` library now part of Ahab, and not Orca?  There may be some more code re-work required for those library functions too.

**Comment by hinchliff** [comment]
@anriban Thanks for fixing the issue with PriorityNetwork.  I see that the code now compiles.

I checked with @paul-ruth, and he confirmed that `orca.ndllib` is the old predecessor of Ahab, and that we should probably update the code to fully use Ahab and not any of the `orca.ndllib` libraries.

Do you want to tackle that too, before we close this ticket?

**Comment by anriban** [comment]
@hinchliff Yes, that is a good idea to migrate some of the old code to use ahab instead of orca.ndllib. I will be working on it. I will update this issue when I am done with that.
